include/tudat/astro/observation_models/corrections/firstOrderRelativisticCorrection.h

```diff
@@ -53,17 +53,19 @@ class FirstOrderLightTimeCorrectionCalculator : public LightTimeCorrection
      *  \param receivingBody Name of receiving body
      *  \param ppnParameterGammaFunction Function returning the parametric post-Newtonian parameter gamma, a measure
      *  for the space-time curvature due to a unit rest mass (default 1.0; value from GR)
+     *  \param bending Boolean flag to determine if light bending should be included.
      */
     FirstOrderLightTimeCorrectionCalculator(
             const std::vector< std::function< Eigen::Vector6d( const double ) > >& perturbingBodyStateFunctions,
             const std::vector< std::function< double( ) > >& perturbingBodyGravitationalParameterFunctions,
             const std::vector< std::string > perturbingBodyNames,
             const std::string transmittingBody,
             const std::string receivingBody,
-            const std::function< double( ) >& ppnParameterGammaFunction = []( ) { return 1.0; } ):
+            const std::function< double( ) >& ppnParameterGammaFunction = []( ) { return 1.0; },
+            const bool bending = true ):
```

Comment:
small point of notation: please modify this to 'useBending' (here and elsewhere)